static/src/images/global/market-up.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="15" height="11" viewBox="0 0 15 11"><path fill="#298422" d="M0 9.97L1.03 11H14l1-1L8 0h-.958L0 9.984z"/></svg>
```

Comment:
Maybe it is just about my skillset (or lack thereof), but server side rendering is so much easier to build and debug (partly because you can use all the abstractions we build).

I don't have docs (i.e. I have no proof), but one of the reasons we switched to the server side rendering was about client side performance too.